ts/Dashboards/EditMode/EditRenderer.ts

```diff
@@ -486,6 +482,14 @@ function renderText(
             {},
             parentElement
         );
+
+        if (options.click) {
+            textElem.addEventListener('click', (): void => {
+                if (options.click instanceof Function) {
+                    options.click();
```

Comment:
When the toggle is created, the click event is also added. So, the click event will be triggered twice on the toggle button.

Reply:
Tested this and everything works the same as in `dashboards/main`. 
When I tried to simplify it the toggle in the sidebar fail. I guess it needs to stay as it is.